### PR TITLE
Added new cmd to remove triggers

### PR DESCRIPTION
**Pull request by Dimedrolity**
There is a problem - one of teams of Moira users have unused triggers. So it is a trash in database. Added command to remove triggers which have ID starting with prefix.

### CODE REVIEW

File: database/redis/trigger_test.go [hunk 1]
```diff
@@ -487,7 +487,7 @@ func TestTriggerStoring(t *testing.T) {
 			So(actual, ShouldResemble, []string{trigger.ID})
 
 			// Now update trigger
-			trigger = &triggers[1]
+			trigger = &testTriggers[1]
```
Comment (kissken) [inline database/redis/trigger_test.go:490]: why?

File: cmd/cli/triggers.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+)
+
+func handleRemoveTriggersByPrefix(logger moira.Logger, database moira.Database, prefix string) error {
+	triggers, err := database.GetTriggerIDsByPrefix(prefix)
+	if err != nil {
+		return err
+	}
+
+	// Added delay because command is potentially dangerous and can delete unwanted triggers
+	const delay = 10 * time.Second
```
Comment (kissken) [inline cmd/cli/triggers.go:16]: let's define outside of the function and left comment there

File: cmd/cli/triggers.go [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	logger.Infof("%d triggers by prefix %s would be removed after %s", len(triggers), prefix, delay)
+	logger.Info("You can cancel execution by Ctrl+C")
+	time.Sleep(delay)
+
+	logger.Infof("Removing triggers by prefix %s started", prefix)
+	for _, id := range triggers {
```
Comment (kissken) [inline cmd/cli/triggers.go:22]: pipe?

File: cmd/cli/triggers.go [hunk 4]
```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"time"
+
+	"github.com/moira-alert/moira"
+)
+
+func handleRemoveTriggersByPrefix(logger moira.Logger, database moira.Database, prefix string) error {
+	triggers, err := database.GetTriggerIDsByPrefix(prefix)
```
Comment (kissken) [inline cmd/cli/triggers.go:10]: does func return triggers by prefix in id, doesnt it?

File: cmd/cli/triggers.go [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+	const delay = 10 * time.Second
+	logger.Infof("%d triggers by prefix %s would be removed after %s", len(triggers), prefix, delay)
+	logger.Info("You can cancel execution by Ctrl+C")
+	time.Sleep(delay)
+
+	logger.Infof("Removing triggers by prefix %s started", prefix)
+	for _, id := range triggers {
+		err := database.RemoveTrigger(id)
+		if err != nil {
+			return err
```
Comment (kissken) [inline cmd/cli/triggers.go:25]: maybe better wrap err?

**Review by kissken — COMMENTED** [review]
left some comments

File: database/redis/trigger_test.go (hunk 1 above)
Comment (kissken) [reply database/redis/trigger_test.go:490]: we already it test file

File: cmd/cli/triggers.go (hunk 4 above)
Comment (Dimedrolity) [reply cmd/cli/triggers.go:10]: yes, noted in [comment](https://github.com/moira-alert/moira/pull/790/commits/ecd22470e6d393f0cfa7b53fbee551a53c199231#diff-f463a96da8e66f155de89be8fc4d0ecde2eb0f6e556a4cc5642b0d3c0e5df5deR153)

File: cmd/cli/triggers.go (hunk 5 above)
Comment (Dimedrolity) [reply cmd/cli/triggers.go:25]: yes, I will do, because there are 2 `return err` cases

File: cmd/cli/triggers.go (hunk 4 above)
Comment (kissken) [reply cmd/cli/triggers.go:10]: think that can be misunderstanding, lets think about naming, cause notion prefix in Moira already use, ya?

File: database/redis/trigger_test.go (hunk 1 above)
Comment (Dimedrolity) [reply database/redis/trigger_test.go:490]: please read the [commit message](https://github.com/moira-alert/moira/pull/790/commits/b3d00e6d69a800be314e00f8b1423e1b4e25a988)

Comment (kissken) [reply database/redis/trigger_test.go:490]: ah, missed this line, ok

File: cmd/cli/triggers.go (hunk 2 above)
Comment (Dimedrolity) [reply cmd/cli/triggers.go:16]: Why? This const is only for one function, so it is local const

File: cmd/cli/triggers.go (hunk 3 above)
Comment (Dimedrolity) [reply cmd/cli/triggers.go:22]: from the [docs](https://redis.io/docs/manual/pipelining/): pipeline is used for improving performance. So pipe for this case is premature optimization, command works pretty fast

File: cmd/cli/triggers.go (hunk 5 above)
Comment (Dimedrolity) [reply cmd/cli/triggers.go]: added both, [first ](https://github.com/moira-alert/moira/pull/790/commits/de53e6d0e9653797804aaec24236aa1765c25897#diff-bbe74c5f7c7c4e46d51dec5dfa5561be8b5fd6f26ae4ea6eac8bc27876f7d98bR13) and [second](https://github.com/moira-alert/moira/pull/790/commits/de53e6d0e9653797804aaec24236aa1765c25897#diff-bbe74c5f7c7c4e46d51dec5dfa5561be8b5fd6f26ae4ea6eac8bc27876f7d98bR26)

File: cmd/cli/triggers.go (hunk 4 above)
Comment (Dimedrolity) [reply cmd/cli/triggers.go:11]: do you mean prefix tree?

Comment (kissken) [reply cmd/cli/triggers.go:11]: yep, as least

Comment (Dimedrolity) [reply cmd/cli/triggers.go:11]: I don't agree about misunderstanding, prefix is common term for programming. So prefix means some symbols at start of string. There is no reference to tree in method name or comment of GetTriggerIDsByPrefix

Comment (Dimedrolity) [reply cmd/cli/triggers.go]: renamed for GetTriggerIDsStartWith(prefix string)

**Review by kissken — APPROVED** [review]
Let’s make release, but rolling with Mikhail (he has rights to do this)
and with Alexey
and maybe ask to do backup before run

**Pull request merged**